src/decision/include/GpsDecision.h

```diff
@@ -14,15 +14,51 @@
 #include <geometry_msgs/Twist.h>
 #include <geometry_msgs/Point.h>
 #include <ros/ros.h>
+#include <std_msgs/Float32.h>
 
 class GpsDecision {
 public:
     GpsDecision(int argc, char **argv, std::string node_name);
-private:
+
+    /**caclutes the distance between wayPoint and current point
+     * @ parameter: given the gps locaiton of the next move
+     * @ return: the distance between the next location and the current location
+     **/
+    static double distance(const geometry_msgs::Point::ConstPtr& relative_gps);
+
+    /**
+     * @ parameter
+     * relative_gps: given the gps locaiton of the next move
+     * current_heading: the curren heading relative to north in degrees (0 to 360 degrees)
```

Comment:
Also, it's fine for this code, but generally we use radians rather then degrees, just for future reference.